content/projects/index.adoc

```diff
@@ -121,6 +121,20 @@ Jenkins X brings along a selection of excellent pre-integrated open source tools
 To use Jenkins X you don't need to install Jenkins separately.
 
 link:jenkins-x[*Learn more*]
+  
+---
+
+== Jenkins Configuration as Code
+
+image:/images/jcasc-logo.png["Jenkins Configuration as Code", role=right]
+
+Setting up Jenkins is a complex process, as both Jenkins and its plugins require some tuning and configuration, 
+with dozens of parameters to set within the web UI manage section.
+
```

Comment:
Comparison with groovy scripts sends a negative "we are better than.." message. We should have this introduction to only focus on what we provide, in positive terms.

proposal:
> Jenkins Configuration as Code allows to define this whole configuration as a simple, human-friendly, plain text yaml syntax. Without any manual step this configuration can be validated and applied to a Jenkins master in fully a reproducible way. With JCasC setting up a new Jenkins master will become a no-brainer event.

Reply:
done!